### PR TITLE
Update es.js

### PR DESCRIPTION
**Pull request by xlNightmare**
All done, and ready to merge

### CODE REVIEW

**Comment by mty22** [comment]
KR prepped, pending review+merge.

**Comment by mty22** [comment]
I've applied a fix, can you please review your respective language and implement? https://github.com/mty22/krunker.io-translations/commit/5ec0f43d446e618fd7d0a446ba2fbcb9138eacd4

**Comment by mty22** [comment]
merge conflict fixed.

**Comment by mty22** [comment]
Still needs a review =(

File: es.js [hunk 1]
```diff
@@ -538,16 +538,16 @@ windows.controls.streak1 Racha 2
...
-windows.controls.propRand Select Random Prop
-windows.controls.propRot Toggle Rotation
-windows.controls.propRotR Reset Rotation
+windows.controls.taunt0 Gesto (Psst)
+windows.controls.taunt1 Gesto (Chiflido)
+windows.controls.taunt2 Gesto (Pedo)
+windows.controls.taunt3 Gesto (Meme)
+windows.controls.taunt4 Gesto (WTF es esto?)
+windows.controls.taunt5 Gesto (Ahora mismo!)
+windows.controls.prop Seleccinar Prop (Mantener Presionado)
```
Comment (1opez) [inline es.js:547]: Selecionar Prop (Mantener Presionado)

File: es.js [hunk 2]
```diff
@@ -173,7 +173,7 @@ settings.interface.dmgCol Color de daño
 settings.interface.critCol Color de daño crítico
 settings.interface.header Interface
 settings.interface.scale Escala de interfaz
-settings.interface.theme Game Theme
+settings.interface.theme Tema
```
Comment (1opez) [inline es.js:176]: Tema del Juego

**Review by 1opez — CHANGES_REQUESTED** [review]

File: es.js (hunk 2 above)
Comment (xlNightmare) [reply es.js:176]: Hey, i noticed your translations are a bit confusing, not really inclusive, so i'll just try and delete some of them to resume it aswell as make it easier to apply the changes.

Thanks a lot for your work

- Nightz

**Comment by mty22** [comment]
@1opez can you please let me know your IGN by pinging me in Krunker Bunker

**Pull request merged**